fixtures-service/json_helpers.py

```diff
@@ -0,0 +1,15 @@
+import json
+import os.path
+
+
+def load(filename):
+    with open(filename) as file:
+        data = json.load(file)
+    return data
```

Comment:
shouldn't the `return` statement be indented under the `with` context manager ? Frankly this is something I'm unsure about. Is there any pros / cons ?

Reply:
No idea. It works and it's not production code. I would leave it as it is.